src/styles/main.scss

```diff
@@ -1 +1,79 @@
-// put code here
+body {
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+}
+
+
+.watch {
+  box-sizing: border-box;
+  width: 80vmin;
+  height: 80vmin;
+  border-radius: 50%;
+  border: 3px dashed black;
+  position: relative;
+
+  &__center {
+    width: 30px;
+    height: 30px;
+    background-color: orange;
+    position: absolute;
+    border-radius: 50%;
+    inset: 0;
+    margin: auto;
+  }
+
+  &__hour-hand {
+    width: 9px;
+    height: 100%;
+    background-color: blue;
+    position: absolute;
+    left: 0;
+    right: 0;
+    margin: auto;
+    animation: rotate-hour-hand 55s linear infinite;
```

Comment:
still not resolved
![image](https://github.com/mate-academy/layout_stop-watch/assets/55200530/74f86b46-feb1-42da-8369-adf150d92a24)